docs/modding/suitmodding/addingblobs.md

```diff
@@ -0,0 +1,253 @@
+---
+sidebar_position: 3
+description: Modifying an image in the menu
+
+---
+# Adding "Blobs"
```

Comment:
Quotes are not necessary, blobs are what they are called

Reply:
I did that to introduce an unfamiliar term. I'll change it